documentation/how-to/pictures.en.md

```diff
@@ -66,6 +66,12 @@ Set the widget type of the field path to *Attachment* and add it to the form
 
 !![widgets](../assets/images/add-1-n-pictures-widgets_picture.png)
 
+## Drawing and Sketching Functionality
+
+QField now has an in-app drawing and sketching functionality, this feature enables to sketch directly over images taken within the application, enhancing data annotation capabilities. The functionality is seamlessly integrated into the attachment widget, allowing users to quickly annotate images with sketches and providing an option to undo sketches if needed for corrections 😉.
```

Comment:
I don't think we need to mention the undo functionality; it's self-evident in the screencast, and I think it's something people would expect in any case :)

```suggestion
QField now has an in-app drawing and sketching functionality enabling you to directly sketch over and annotate images captured while in the field as well as drawing on top of a blank canvas or over a template. The functionality is seamlessly integrated into the feature form's attachment widget.
```

Reply:
Should we drop the "now"? This functionality will soon not be so new anymore

Reply:
@m-kuhn , yep.